src/app/Models/constants.ts

```diff
@@ -7,3 +7,11 @@ export const FACEBOOKICON = '../../../assets/icons/facebook-icon.svg';
 export const TWITTERICON = '../../../assets/icons/twitter-icon.svg';
 export const INSTRAGRAMICON = '../../../assets/icons/instragram-icon.svg';
 export const YOUTUBEICON = '../../../assets/icons/youtube-icon.svg';
+export let BLOGTAGS: string[] = [
+  'Technology',
+  'Poetry',
+  'Films',
+  'World politics',
+];
+export const DUMMYUSERIMAGE =
```

Comment:
constant names are NOTREADABLEATALL, we can use underscore between words. thank you